src/test/java/com/cloudbees/jenkins/support/impl/ProxyConfigurationTest.java

```diff
@@ -0,0 +1,47 @@
+package com.cloudbees.jenkins.support.impl;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import com.cloudbees.jenkins.support.SupportTestUtils;
+import com.cloudbees.jenkins.support.api.Component;
+import hudson.ExtensionList;
+import hudson.ProxyConfiguration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class ProxyConfigurationTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @Test
+    @Issue({"JENKINS-68008", "JENKINS-73599"})
+    public void testProxyContent() {
+        List<String> noProxyHosts = Arrays.asList(".server.com", "*.example.com");
+        j.jenkins.setProxy(new ProxyConfiguration(
+                "proxy.server.com",
+                1234,
+                "proxyUser",
+                "proxyPass",
+                String.join("\n", noProxyHosts),
+                "http://localhost:8080"));
+
+        String ucMdToString =
+                SupportTestUtils.invokeComponentToString(Objects.requireNonNull(ExtensionList.lookup(Component.class)
+                        .get(com.cloudbees.jenkins.support.impl.ProxyConfiguration.class)));
+        assertThat(ucMdToString, containsString(" - Host: `proxy.server.com`"));
```

Comment:
Perhaps off topic, but have we considered throwing out the convention of formatting files in support bundles using Markdown, and switching to YAML?

Reply:
@PierreBtz we can certainly make such a change in the plugin, but does @cloudbees have some internal tool which parses the current text files? If so, it would need to be modified (simplified, I hope).

Reply:
Yes we do... and I don't like parsing markdown :)